build.gradle

```diff
@@ -105,6 +105,7 @@ allprojects {
   repositories {
     jcenter()
     mavenCentral()
+    maven { url "https://repository.apache.org/content/repositories/orgapachetuweni-1020" }
```

Comment:
Is this just needed until the 2.0.0 release is approved?

Reply:
Indeed. This is the staged repository for the current RC.